wp-includes/functions.php

```diff
@@ -5736,19 +5736,36 @@ function wp_validate_boolean( $var ) {
 /**
  * Delete a file
  *
- * @since 4.2.0
+ * @since 4.9.2
  *
- * @param string $file The path to the file to delete.
+ * @param string $file                The path to the file to delete.
+ * @param array  $directories_unblock Are directory to unblocked. They are ROOT, WP_CONTENT_DIR, WP_ADMIN_DIR
+ *                                   and WP_INCLUDE_DIR.
+ * @return bool Whether the param is invalidated.
```

Comment:
Do you mean: "Whether the file has been deleted" ?

If so, the function would need to return the result of the `unlink()` call and it would need an additional `return false;` at the very end.

Otherwise the return value is inconsistent.

Reply:
you right, but i don't need return false, only return is ok. done.

Reply:
@lenonleite By not returning `true` when the delete has been executed, you make the return value inconsistent and unreliable, so please reconsider.